.travis.yml

```diff
@@ -5,10 +5,11 @@ python:
   - "3.8"
 install:
   - pip install --upgrade pip
-  - pip install numpy jaxlib tensorflow tensorflow-datasets --upgrade
+  - pip install pygame==2.0.0.dev6 tfp-nightly tfds-nightly numpy jaxlib jax tf-nightly matplotlib more-itertools --upgrade
   - pip install git+https://github.com/google/jax.git
   - pip install -e .
 script:
   - set -e
-  - for f in tests/*.py; do python $f; done
-
+  # - for f in tests/*.py; do python $f; done
```

Comment:
1) In these kind of places, perhaps add a TODO comment above + quick note on what's failing and why, to be able to find them later?

2) Do you know if all tests are failing, or just some subset of them? I.e. you could, optionally call the passing test files individually here.

Reply:
@romanngg Roman, for test files except `infinite_fcn_test`, `function_space_test`, `weight_space_test` and `stax_test`, I have not replaced the JAX support with TF support (a lot of workload, and probably not be able to do it before the deadline). I will add TODO comments in Travis CI.

Reply:
Great! In that case I suggest to just call the tests that pass here individually.

Reply:
Sure!

Reply:
@romanngg Just to make sure, call the test files individually rather than test cases individually, if I understand correctly?

Reply:
Yes, I think test cases would be too much hassle

Reply:
Got it! Thanks, Roman!